src/EmailPage/index.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+import * as React from 'react';
+import css from 'styled-jsx/css';
+import { Close } from '@kiwicom/orbit-components/lib/icons';
+import image from '../../static/mailbox@2x.png';
+
+type Props = {
+  email: string,
+};
+
+const style = css`
+  .Email {
+    position: absolute;
```

Comment:
`position: absolute`, right, background-color, height and box-shadow are not required anymore since each component inherits this attributes from `Layout`.